docs/Learning-Environment-Design-Agents.md

```diff
@@ -647,6 +649,29 @@ Notes:
 - You cannot mask all the actions of a branch.
 - You cannot mask actions in continuous control.
 
+
+#### IActuator interface and ActuatorComponents
```

Comment:
```suggestion
### IActuator interface and ActuatorComponents
```
(same level as `Discrete Actions`)

Reply:
Arg, couldn't get the multi-line suggestion working, but add a line for this under Discrete Actions in the ToC